save-common/src/commonMain/kotlin/org/cqfn/save/core/logging/Logger.kt

```diff
@@ -16,13 +18,24 @@ import kotlinx.datetime.toLocalDateTime
 /**
  *  Logging mode
  */
-var logType: LogType = LogType.WARN
+expect var logType: GenericAtomicReference<LogType>
```

Comment:
Can it be ordinary var, created using `GenericAtomicReference`'s `expect constructor`?

Reply:
ok, that might look nicer